monitor/linuxmonitor/cgnetcls/cgnetcls.go

```diff
@@ -213,6 +219,19 @@ func mountCgroupController() {
 
 }
 
+// CgroupMemberCount -- Gives the cound of the number of processes in a cgroup
```

Comment:
this is not a valid go comment. The comments are used by godoc to generate a document. It must be plain english `CgroupMemberCount returns ...`